test/conformance/runtime/cgroup_test.go

```diff
@@ -41,6 +42,16 @@ func toMilliValue(value float64) string {
 	return fmt.Sprintf("%dm", int(value*1000))
 }
 
+func isCgroupsV2(t *testing.T, mounts []*types.Mount) bool {
+	for _, mount := range mounts {
+		if mount.Path == "/sys/fs/cgroup" {
+			return mount.Type == "cgroup2"
+		}
+	}
+	t.Fatal("Failed to find cgroup mount on /sys/fs/cgroup")
```

Comment:
Can this ever return false? I think it fail the test with CGroups v1 and that's probably not wanted.

Reply:
Otherwise this PR looks good to me.

Reply:
Cgroup v1 also has the mount path `"/sys/fs/cgroup"` but it is not `cgorup2` mount type. So cgroup v1 returns false at L#48.

```go
		if mount.Path == "/sys/fs/cgroup" {
			return mount.Type == "cgroup2"
		}
```

`return false` at L#52 is unreachable because `t.Fatal` but we need to define it otherwise it fails to build.

Reply:
OK. Thanks. In that case I'd probably return an error along with bool to make it more clear:
```
func isCgroupsV2(mounts []*types.Mount) (bool, error) {
	for _, mount := range mounts {
		if mount.Path == "/sys/fs/cgroup" {
			return mount.Type == "cgroup2", nil
		}
	}
        return false, fmt.Errorf("failed to find cgroup mount on /sys/fs/cgroup")
}
```